developer_manual/core/theming.rst

```diff
@@ -1,166 +1,183 @@
+   sure you change any references to match the location of your owncloud
+   installation.
+
+How to Create a New Theme
+-------------------------
+
+At its most basic, to create a theme requires two steps:
+
+1. Copy and extend an existing theme or create one from scratch.
+2. Enable the theme in the ownCloud Admin dashboard.
+
+All themes, whether copied or new, must meet two key criteria, these are:
+
+1. They must be located in a directory under the ``apps`` directory of your ownCloud installation.
+2. They require a configuration file called ``appinfo/info.xml`` to be present.
+
+appinfo/info.xml
+~~~~~~~~~~~~~~~~
+
+Here’s an example of the bare minimum which the file needs to contain: 
+
+::
+
+  <?xml version="1.0"?>
+  <info>
+      <id>theme-example</id>
+      <name>Enterprise ownCloud Theme</name>
+      <types>
+          <theme/>
+      </types>
+      <dependencies>
+        <owncloud min-version="9.2" max-version="10" />
+      </dependencies>
+  </info>
+
+And here’s a longer, more complete example:
+
+::
+
+  <?xml version="1.0"?>
+  <info>
+      <id>theme-example</id>
+      <name>Enterprise ownCloud Theme</name>
+      <description>This App provides the enterprise ownCloud theme.</description>
+      <licence>AGPL</licence>
+      <author>Philipp Schaffrath</author>
+      <version>0.0.1</version>
+      <types>
+          <theme/>
+      </types>
+      <dependencies>
+          <owncloud min-version="9.2" max-version="10" />
+      </dependencies>
+  </info>
+
+The value of the ``id`` element needs to be the name of your theme’s folder. We recommend that it always be prefixed with ``theme-``. The main reason for doing so, is that when being viewed in the ownCloud Admin dashboard, it's clear that it’s a theme, not an application. 
+
+The ``type`` element needs to be the same as is listed above.
+The dependencies element needs to be present to set the minimum and maximum versions of ownCloud which are supported. If it’s not present, a warning will be displayed in ownCloud 10 and an error will be thrown in the upcoming ownCloud 11.
+
+While the remaining elements are optional, they help when working with the theme in the ownCloud Admin dashboard. 
+Please consider filling out as many as possible, as completely as possible.
+
+How to Override Images
+----------------------
+
+Any image, such as the default logo, can be overridden by including one with the same path structure in your theme.
+For example, let’s say that you want to replace the logo on the login-page above the credentials-box which, by default is has the path: ``owncloud/core/img/logo-icon.svg``.
+To override it, assuming that your custom theme was called ``theme-example`` (*which will be assumed for the remainder of the theming documentation*), add a new file with the following path: ``owncloud/apps/theme-example/img/logo-icon.svg``.
+After the theme is activated, this image will override the default one.
+
+.. note::
```

Comment:
I don't mind this section, but actually we don't do anything special for those formats. It is just what modern browsers support. So we would support gif and other formats too.